website/screens/components/toggle-group/specs/ToggleGroupSpecsPage.tsx

```diff
@@ -0,0 +1,641 @@
+import { DxcTable, DxcStack, DxcText, DxcList } from "@dxc-technology/halstack-react";
+import Image from "@/common/Image";
+import HeadingLink from "../../../common/HeadingLink";
+import toggleGroupAnatomy from "./images/toggle_group_anatomy.png";
+import toggleGroupStates from "./images/toggle_group_states.png";
+import toggleGroupSpecs from "./images/toggle_group_specs.png";
+import Figure from "../../../common/Figure";
+import Code from "../../../common/Code";
+import DocFooter from "../../../common/DocFooter";
+
+const ToggleGroupSpecsPage = () => {
+  return (
+    <DxcStack gutter="xxxlarge">
+      <DxcStack gutter="large">
+        <HeadingLink level={2}>Specifications</HeadingLink>
+        <Figure caption="Toggle design specifications">
+          <Image src={toggleGroupSpecs} alt="Toggle design specifications" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Anatomy</HeadingLink>
+        <Image src={toggleGroupAnatomy} alt="Toggle design specifications" />
+        <DxcList type="number">
+          <DxcText>Label</DxcText>
+          <DxcText>Helper text</DxcText>
+          <DxcText>Container</DxcText>
+          <DxcText>Button</DxcText>
+          <DxcText>Button icon</DxcText>
+          <DxcText>Button label</DxcText>
+        </DxcList>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>States</HeadingLink>
+        <DxcText as="p">
+          Different states are defined in the life cycle of the component:
+          <strong>Unselected enabled</strong>, <strong>unselected hover</strong>,{" "}
+          <strong>unselected focus</strong>, <strong>unselected active</strong>,{" "}
+          <strong>unselected disabled</strong>, <strong>selected enabled</strong>,{" "}
+          <strong>selected hover</strong>, <strong>selected focus</strong>,{" "}
+          <strong>selected active</strong> and <strong>selected disabled</strong>
+        </DxcText>
+        <Figure caption="Toggle button states example">
+          <Image src={toggleGroupStates} alt="Toggle button states example" />
+        </Figure>
+      </DxcStack>
+      <DxcStack gutter="large">
```

Comment:
The order should be the same as in the markdown and in the documentation of the rest of the components. First, the `States` section and then the `Anatomy` one.